backend/app/views/spree/admin/promotions/index.html.erb

```diff
@@ -85,11 +77,26 @@
       <% @promotions.each do |promotion| %>
         <tr id="<%= spree_dom_id promotion %>" class="<%= cycle('odd', 'even')%>">
           <td><%= promotion.name %></td>
-          <td class="align-center"><%= (promotion.codes.size == 1) ? promotion.codes.first.try!(:value) : Spree.t(:number_of_codes, count: promotion.codes.size) %></td>
-          <td><%= promotion.description %></td>
-          <td class="align-center"><%= promotion.usage_limit.nil? ? "∞" : promotion.usage_limit  %></td>
-          <td class="align-center"><%= Spree.t(:current_promotion_usage, count: promotion.usage_count) %></td>
-          <td class="align-center"><%= promotion.expires_at.to_date.to_s(:short_date) if promotion.expires_at %></td>
+          <td class="align-center">
+            <%= (promotion.codes.size == 1) ? promotion.codes.first.try!(:value) : Spree.t(:number_of_codes, count: promotion.codes.size) %>
+          </td>
+          <td class="align-center">
+            <span class="pill pill-<%= promotion.active? ? 'active' : 'inactive' %>">
+              <%= Spree.t(promotion.active?? :active : :inactive) %>
+            </span>
+          </td>
+          <td class="align-center">
+            <%= promotion.usage_limit.nil? ? "∞" : promotion.usage_limit %>
+          </td>
+          <td class="align-center">
+            <%= promotion.usage_count %>
+          </td>
+          <td class="align-center">
+            <%= promotion.starts_at.to_date.to_s(:short_date) if promotion.starts_at %>
+          </td>
+          <td class="align-center">
+            <%= promotion.expires_at.to_date.to_s(:short_date) if promotion.expires_at %>
+          </td>
```

Comment:
You are changing the table far more than just adding the pill component. Could you please do this in a separate commit, or not even PR?